app/models/article.js

```diff
@@ -0,0 +1,18 @@
+// Example model
+
+var mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
```

Comment:
Better use ES6 imports consistently. This should also be marked as error by our eslintrc (use of var, and multiple instructions wihout semicolon)...